contrib/init.d/pbs_mom.in

```diff
@@ -28,12 +29,30 @@ else
 fi
 
 pidof_pbs_mom() {
-	pid=`pidof pbs_mom`
-	if [ $? -eq 0 ]; then
-		echo $pid
-	else
-		echo "-1"
-	fi
+    pids=`pidof pbs_mom`
+    if [ $? -ne 0 ]; then
+        echo "-1"
```

Comment:
i'd put this one line to indicate it's sort of an error condition.
it would be betetr though if you just did return  and no echo, and used `$?` to check if it was succesful in the code below (instead of comparing it with `-1`).